pkg/output/output.go

```diff
@@ -36,6 +36,7 @@ import (
 	"github.com/edoardottt/cariddi/pkg/scanner"
 )
 
+// constant defined in file.go as well, redefining here for circular dependency.
```

Comment:
Isn't possible to define just one constant somewhere (file / output) and import that?

Reply:
Updated the PR. This comment was incorrect, constant is defined just once so removed the comment.